### ISSUE TITLE
get a error  "could not build Objective-C module 'QueryKit' "  After upgrading xcode7

### ISSUE THREAD

**Issue by since66** [QueryKit/QueryKit#46]
/Pods/QueryKit/QueryKit/QueryKit.h:10:9: error: include of non-modular header inside framework module 'QueryKit.QueryKit'
#import <QueryKit/QKAttribute.h>

/Pods/QueryKit/QueryKit/QueryKit.h:11:9: error: include of non-modular header inside framework module 'QueryKit.QueryKit'
#import <QueryKit/QKQuerySet.h>
        ^
<unknown>:0: error: could not build Objective-C module 'QueryKit'

How to fix the error？

**Comment by wodetian** [comment]
OK  ,You need to modify `QueryKit.h` files 

#45 
````
#import <Foundation/Foundation.h>

//! Project version number for QueryKit.
FOUNDATION_EXPORT double QueryKitVersionNumber;

//! Project version string for QueryKit.
FOUNDATION_EXPORT const unsigned char QueryKitVersionString[];

// In this header, you should import all the public headers of your framework using statements like #import <QueryKit/PublicHeader.h>
#import "QKAttribute.h"
#import "QKQuerySet.h"
```

**Comment by wodetian** [comment]
#46

**Comment by kostiakoval** [comment]
@since66  Quick Fix:  remove all content of the `QueryKit.h` but keep the file, don't remove it from project.

**Comment by kylef** [comment]
I've released QueryKit 0.12.0 which solves this problem. Thanks for reporting!
